src/fipp/edn.cljc

```diff
@@ -38,18 +55,41 @@
   (visit-seq [this x]
     (if-let [pretty (symbols (first x))]
       (pretty this x)
-      [:group "(" [:align (interpose :line (map #(visit this %) x))] ")"]))
+      (let [printer (cond-> this
```

Comment:
This pattern repeats for seqs, vectors, maps, and sets. Could it be abstracted in to a `print-coll` function?